community/modules/compute/notebook/metadata.yaml

```diff
@@ -18,3 +18,10 @@ spec:
     services:
     - notebooks.googleapis.com
     - storage.googleapis.com
+ghpc:
+  validations:
```

Comment:
you should align this with the blueprint language. Make sure you have the same terms whenever possible